### PR TITLE
⬆️ Update to Null Safety and Android V2 embedding

### LINKED ISSUE #8: Null-safety support?
Are there plans to support null-safety?
Thanks!

### PR DESCRIPTION
**Pull request by Inventrohyder**
Closes #8

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 name: scrolling_page_indicator
 description: Flutter implementation of page view indicator like Instagram
-version: 0.1.3
-author: "Nikita Zhelonkin <niks.mbox@gmail.com>"
```
Comment (NikitaZhelonkin) [inline pubspec.yaml:4]: ?

File: lib/scrolling_page_indicator.dart [hunk 2]
```diff
@@ -184,24 +183,27 @@ class _Painter extends CustomPainter {
...
           if (dot - _visibleFramePosition < currentScaleDistance) {
-            double calculatedDiameter = diameter * (dot - _visibleFramePosition) /
-                currentScaleDistance;
-            diameter = min(diameter, calculatedDiameter);
-          } else if (dot - _visibleFramePosition > width - currentScaleDistance) {
             double calculatedDiameter =
-                diameter * (-dot + _visibleFramePosition + width) / currentScaleDistance;
+                diameter * (dot - _visibleFramePosition) / currentScaleDistance;
+            diameter = min(diameter, calculatedDiameter);
+          } else if (dot - _visibleFramePosition >
```
Comment (NikitaZhelonkin) [inline lib/scrolling_page_indicator.dart:192]: Bad formatting. Make line hard wrap at least 100-120 columns in your ide and then do reformat code.

**Review by NikitaZhelonkin — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (Inventrohyder) [reply pubspec.yaml:4]: `author` is deprecated, use verified publishers instead

https://stackoverflow.com/a/59942840

File: lib/scrolling_page_indicator.dart (hunk 2 above)
Comment (Inventrohyder) [reply lib/scrolling_page_indicator.dart:192]: You can check it again, I've reformatted

**Comment by Inventrohyder** [comment]
@NikitaZhelonkin Ready for another pass through

**Review by NikitaZhelonkin — APPROVED** [review]

**Pull request merged**